src/platform/ESP32/nimble/BLEManagerImpl.cpp

```diff
@@ -470,7 +470,7 @@ void BLEManagerImpl::DriveBLEState(void)
         if (GetFlag(mFlags, kFlag_Advertising))
         {
             err = ble_gap_adv_stop();
-            if (err != CHIP_NO_ERROR)
+            if (err != 0)
```

Comment:
This change doesn't really make sense.  If the point is that the value returned by `ble_gap_adv_stop` is not a `CHIP_ERROR`.... well, we're assigning it to a CHIP_ERROR.  Maybe we should have a separate variable for the non-CHIP-error return values in this function?

Doing that in #4256 is fine.

Reply:
Ohh, my bad. I totally missed doing that. I have updated the PR with the change. Thanks!

Reply:
Ohh. I totally missed doing that. I have updated the PR with the changes. Also, made similar change in `StartAdvertising` to not use `CHIP_ERROR` type variable for ESP32 NimBLE APIs. PTAL